platforms/access-bridge/src/services/stripe-service.ts

```diff
@@ -76,4 +78,50 @@ export class StripeService {
       throw e;
     }
   }
+
+  /**
+   * Creates a Stripe Checkout session based on the provided price ID.
+   * @param viewer Email address and viewer id from the auth token used for creating the checkout session.
+   * @param params Stripe checkout params to use for creating the checkout session.
+   * @returns A Promise resolving to a Stripe Checkout Session object, including a URL for the checkout page.
+   * @throws Error if there is an issue creating the checkout session or if the price ID is invalid.
+   */
+  async createCheckoutSession(viewer: Viewer, params: StripeCheckoutParams): Promise<Stripe.Checkout.Session> {
+    try {
+      const session = await this.stripe.checkout.sessions.create({
+        payment_method_types: ['card'],
+        line_items: [
+          {
+            price: params.price_id,
+            quantity: 1,
+          },
+        ],
+        metadata: {
```

Comment:
`metadata` is just for **single payments**, for **subscriptions**, we need `subscription_metadata` parameter here.